src/lib/features/instance-stats/getProductionChanges.ts

```diff
@@ -0,0 +1,44 @@
+import { type Db } from 'lib/server-impl';
+import { GetActiveUsers } from './getActiveUsers';
+
+export type GetProductionChanges = () => Promise<{
+    last30: number;
+    last60: number;
+    last90: number;
+}>;
+
+export const createGetProductionChanges =
+    (db: Db): GetProductionChanges =>
+    async () => {
+        const productionChanges = await db
+            .select({
+                last_month: db.raw(
+                    "SUM(CASE WHEN day > NOW() - INTERVAL '30 days' THEN updates END)",
+                ),
+                last_two_months: db.raw(
+                    "SUM(CASE WHEN day > NOW() - INTERVAL '60 days' THEN updates END)",
+                ),
+                last_quarter: db.raw(
+                    "SUM(CASE WHEN day > NOW() - INTERVAL '90 days' THEN updates END)",
+                ),
+            })
+            .from('stat_environment_updates');
```

Comment:
I would expect a where clause here, to only pick production envs

Reply:
Totally agree. thanks for the reminder

Reply:
I also updated it to look at the environment type and sum across all environments of type 'production'.